packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -840,4 +824,150 @@ describe('CSFLE / QE', function () {
+            __safeContent__: [
+              Binary.createFromHexString("b04e26633d569cb47b9cbec650d812a597ffdadacb9a61ee7b1661f52228d661", 0)
+            ]
+          }
+        ],
+        bypassDocumentValidation: true
+      });`);
+
+      await browser.shellEval(`db.runCommand({
+        insert: 'enxcol_.${collectionName}.ecoc',
+        documents: [
+          {
+            _id: ObjectId("6454e14689ef42f381f7336b"),
+            fieldName: 'v',
+            value: Binary.createFromHexString("3eb89d3a95cf955ca0c8c56e54018657a45daaf465dd967d9b24895a188d7e3055734f3c0af88302ceab460874f3806fe52fa4541c9f4b32b5cee6c5a6df9399da664f576dd9bde23bce92f5deea0cb3", 0)
+          },
+          {
+            _id: ObjectId("6454e14689ef42f381f73385"),
+            fieldName: 'v',
+            value: Binary.createFromHexString("2299cd805a28efb6503120e0250798f1b19137d8234690d12eb7e3b7fa74edd28e80c26022c00d53f5983f16e7b5abb7c3b95e30f265a7ba36adb290eda39370b30cedba960a4002089eb5de2fd118fc", 0)
+          }
+        ],
+        bypassDocumentValidation: true
+      });`);
+
+      await browser.shellEval(`db.runCommand({
+        insert: 'enxcol_.${collectionName}.esc',
+        documents: [
+          {
+            _id: Binary.createFromHexString("51db700df02cbbfa25498921f858d3a9d5568cabb97f7283e7b3c9d0e3520ac4", 0),
+            value: Binary.createFromHexString("dc7169f28df2c990551b098b8dec8f5b1bfeb65d3f40d0fdb241a518310674a6", 0)
+          },
+          {
+            _id: Binary.createFromHexString("948b3d29e335485b0503ffc6ade6bfa6fce664c2a1d14790a523c09223da3f09", 0),
+            value: Binary.createFromHexString("7622097476c59c0ca5bf9d05a52fe725517e03ad811f6c073b0d0184a9d26131", 0)
+          }
+        ],
+        bypassDocumentValidation: true
+      });`);
+
+      await browser.disconnect();
+
+      // now connect with QE and check that we can query the data stored in a 6.x database
+      await browser.connectWithConnectionForm({
+        hosts: [CONNECTION_HOSTS],
+        fleKeyVaultNamespace: `${databaseName}.keyvault`,
+        fleKey: 'A'.repeat(128),
+      });
+
+      await browser.navigateToCollectionTab(
+        databaseName,
+        collectionName,
+        'Documents'
+      );
+
+      // { v: "123", _id: 'asdf' }
+      // { v: "456", _id: 'ghjk' }
+
+      let decryptedResult = await getFirstListDocument(browser);
+      delete decryptedResult.__safeContent__;
+      expect(decryptedResult).to.deep.equal({ v: '"123"', _id: '"asdf"' });
+
+      await browser.runFindOperation('Documents', '{ v: "456" }');
+
+      decryptedResult = await getFirstListDocument(browser);
+      delete decryptedResult.__safeContent__;
+      expect(decryptedResult).to.deep.equal({ v: '"456"', _id: '"ghjk"' });
```

Comment:
This finds nothing. Testing this manually behaves the same. It does show the docs in the collection and they are decrypted, it seems to just be the equality check that doesn't work. Yet it does work in the equivalent mongosh tests.